src/local/butler/common.py

```diff
@@ -301,6 +301,13 @@ def install_dependencies(platform_name=None, is_reproduce_tool_setup=False):
         'src/third_party',
         platform_name=platform_name)
 
+  with tempfile.NamedTemporaryFile() as f:
+    f.write(open('src/requirements.txt', 'rb').read())
+    f.write(open('src/appengine/requirements.txt', 'rb').read())
+    f.flush()
+
+    _install_pip(f.name, 'src/appengine/third_party')
```

Comment:
nit: why not two _install_pip calls rather than temp file approach.

Reply:
That didn't work for some reason. The second call would overwrite/remove some things in the first call.